app/create_app.py

```diff
@@ -181,26 +198,39 @@ def inject_service_name():
     def inject_content_urls():
         try:
             fund: Fund = find_fund_in_request()
-            if fund:
-                round = find_round_in_request(fund)
-                if round:
-                    return dict(
-                        contact_us_url=url_for(
-                            "content_routes.contact_us",
-                            fund=fund.short_name,
-                            round=round.short_name,
-                        ),
-                        privacy_url=url_for(
-                            "content_routes.privacy",
-                            fund=fund.short_name,
-                            round=round.short_name,
-                        ),
-                        feedback_url=url_for(
-                            "content_routes.feedback",
-                            fund=fund.short_name,
-                            round=round.short_name,
-                        ),
-                    )
+            if not fund:
+                application_id = (
+                    request.view_args.get("application_id")
+                    if request.view_args.get("application_id")
+                    else request.form["application_id"]
+                )
+                application = get_application_data(
+                    application_id, as_dict=True
+                )
+                fund = get_fund_data(
```

Comment:
Should probably check if application is not None here

Reply:
Here I want to catch exception if application is None. so don't want to check